### ISSUE TITLE
[package request] BespokeSynth

### ISSUE THREAD

**Issue by IcedQuinn** [void-linux/void-packages#36424]
https://github.com/BespokeSynth/BespokeSynth

A FOSS modular synthesizer/DAW. Or as the author describes it, "Ableton Live if i smashed it with a bat and asked you to put it back together."

I've compiled this locally and am willing to package/PR it myself if approved. It contains some vendored dependencies and extricating them could be awkward. Music stuff is usually rejected by package management for this reason.

**Comment by notthewave** [comment]
there is probably patching required for musl as it complains about locale stuff

**Comment by IcedQuinn** [comment]
> musl
right. i don't have a musl install on hand to test that with. i know on alpine some of the synths (surge) were a disaster but on glibc distros they work fine.

**Comment by luzpaz** [comment]
soft bump

**Comment by luzpaz** [comment]
1.3.0 has been released (https://github.com/BespokeSynth/BespokeSynth/releases/tag/v1.3.0)
